components/telemetry-operator/api/v1alpha1/logpipeline_types.go

```diff
@@ -147,7 +147,6 @@ func filterOutCondition(conds []LogPipelineCondition, condType LogPipelineCondit
 //+kubebuilder:resource:scope=Cluster
 //+kubebuilder:subresource:status
 //+kubebuilder:printcolumn:name="Status",type=string,JSONPath=`.status.conditions[-1].type`
-//+kubebuilder:printcolumn:name="Unsupported-Plugins",type=boolean,JSONPath=`.spec.enableUnsupportedPlugins`
 //+kubebuilder:printcolumn:name="Age",type=date,JSONPath=`.metadata.creationTimestamp`
 
 // LogPipeline is the Schema for the logpipelines API
```

Comment:
If this was lower case by purpose, ignore my suggestion

```suggestion
// LogPipeline is the Schema for the LogPipelines API
```

Reply:
The lower-case logpipeline refers to the Kubernetes resources. It's correct in this case

Reply:
will leave it as it is